World Crime Rates Analysis/Model/README.md

```diff
@@ -0,0 +1,69 @@
+**PROJECT TITLE**
```

Comment:
You have done a great work, your README file is impressive. But one silly thing is there, in the README file put your project name in the place of Project Title heading.

Reply:
Okay, doing it.